cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -190,24 +253,35 @@ void run_mnist(const std::string& model_name,
                       SRTensorTypeFloat,
                       SRMemLayoutNested);
     double put_tensor_end = MPI_Wtime();
+    log_data(context, LLDebug, "put_tensor completed for rank: " + std::to_string(rank));
     delta_t = put_tensor_end - put_tensor_start;
     put_tensor_times.push_back(delta_t);
 
     double run_script_start = MPI_Wtime();
-    if (use_multigpu)
+    if (use_multigpu) {
       client.run_script_multigpu(script_key, "pre_process_3ch", {in_key}, {script_out_key}, rank, 0, num_devices);
-    else
+      log_data(context, LLDebug, "Use_multigpu=True - script_key: " + script_key + " in_key: " + in_key + "script_out_key: " + script_out_key + " rank: " + std::to_string(rank) + " num_devices: " + std::to_string(num_devices));
```

Comment:
split and check log message length
(General rule: if it's going over a single line, split it into a temp var)